datastore-translator/translator_integration_test.go

```diff
@@ -11,14 +11,21 @@ import (
 	"gotest.tools/assert"
 	"log"
 	"testing"
+	"time"
 )
 
+const DATASTORE_CONNECT_TIMEOUT = 5 * time.Second
+
 func TestIntegration(t *testing.T) {
 	ctx := context.Background()
+
 	// 1. create a new datastore client
 	client, err := datastore.NewClient(ctx, "st2-saas-prototype-dev")
 	assert.NilError(t, err)
 
+	ctx, cancel := context.WithTimeout(ctx, DATASTORE_CONNECT_TIMEOUT)
```

Comment:
NOTE: Per documentation, connection is established lazily on first operation so we want to use this context with timeout for subsequent datastore client operations.

It will have no affect if it's only used for ``datastore.NewClient``.